<?xml version='1.0' encoding='utf-8'?>
<robot name="frontec_arm">
	<group name="leapting_arm">
		<joint name="joint_a1" />
		<joint name="joint_a2" />
		<joint name="joint_a3" />
		<joint name="joint_a4" />
		<joint name="joint_a5" />
		<joint name="joint_a6" />
		<joint name="joint_a6_tool0" />
	</group>
	<group_state name="home_srdf" group="leapting_arm">
		<joint name="joint_a1" value="0" />
		<joint name="joint_a2" value="-1.26484" />
		<joint name="joint_a3" value="0" />
		<joint name="joint_a4" value="0" />
		<joint name="joint_a5" value="0" />
		<joint name="joint_a6" value="0" />
	</group_state>
	<group_state name="armInitPose" group="leapting_arm">
		<joint name="joint_a1" value="0.02880265" />
		<joint name="joint_a2" value="-2.4331412048974546" />
		<joint name="joint_a3" value="2.3829608596009297" />
		<joint name="joint_a4" value="-0.005232654177111679" />
		<joint name="joint_a5" value="1.5335307198314294" />
		<joint name="joint_a6" value="0.022413919679839134" />
	</group_state>
	<group_state name="armTopPoseCart" group="leapting_arm">
		<joint name="joint_a1" value="-0.937960055" />
		<joint name="joint_a2" value="-1.38037085230" />
		<joint name="joint_a3" value="1.2025756972" />
		<joint name="joint_a4" value="-0.02199963087529" />
		<joint name="joint_a5" value="1.330925534" />
		<joint name="joint_a6" value="0.0343809442" />
	</group_state>
	<group_state name="armTopPose" group="leapting_arm">
		<joint name="joint_a1" value="-1.107266591" />
		<joint name="joint_a2" value="-1.3236103713831855" />
		<joint name="joint_a3" value="1.1248573201676095" />
		<joint name="joint_a4" value="-0.014224119244488426" />
		<joint name="joint_a5" value="1.309681509488257" />
		<joint name="joint_a6" value="0.0" />
	</group_state>
	<group_state name="armPickupDetectPose" group="leapting_arm">
		<joint name="joint_a1" value="0.0317965739" />
		<joint name="joint_a2" value="-1.7829942" />
		<joint name="joint_a3" value="1.746261624" />
		<joint name="joint_a4" value="-0.016031180" />
		<joint name="joint_a5" value="1.4997467829" />
		<joint name="joint_a6" value="0.02358802" />
	</group_state>
	<group_state name="armPlaceDetectPose" group="leapting_arm">
		<joint name="joint_a1" value="-1.618277754" />
		<joint name="joint_a2" value="-1.579361844258" />
		<joint name="joint_a3" value="1.12114781482" />
		<joint name="joint_a4" value="0.006897768096684" />
		<joint name="joint_a5" value="1.62371093980" />
		<joint name="joint_a6" value="-1.66032628822" />
	</group_state>
	<group_state name="armTopPoseNeg" group="leapting_arm">
		<joint name="joint_a1" value="-1.107266591" />
		<joint name="joint_a2" value="-1.3236103713831855" />
		<joint name="joint_a3" value="1.1248573201676095" />
		<joint name="joint_a4" value="-0.014224119244488426" />
		<joint name="joint_a5" value="1.309681509488257" />
		<joint name="joint_a6" value="0.0" />
	</group_state>
	<group_state name="armPlaceDetectPoseNeg" group="leapting_arm">
		<joint name="joint_a1" value="-1.618277754" />
		<joint name="joint_a2" value="-1.579361844258" />
		<joint name="joint_a3" value="1.12114781482" />
		<joint name="joint_a4" value="0.006897768096684" />
		<joint name="joint_a5" value="1.62371093980" />
		<joint name="joint_a6" value="-1.66032628822" />
	</group_state>
	<group_state name="armPickupMidPoseNeg" group="leapting_arm">
		<joint name="joint_a1" value="0.30181207999999" />
		<joint name="joint_a2" value="-1.44733110" />
		<joint name="joint_a3" value="1.639219586" />
		<joint name="joint_a4" value="-0.01607554706171" />
		<joint name="joint_a5" value="1.499758" />
		<joint name="joint_a6" value="0.154819815" />
	</group_state>
	<group_state name="armTopPosePos" group="leapting_arm">
		<joint name="joint_a1" value="2.034326009" />
		<joint name="joint_a2" value="-1.3236103713831855" />
		<joint name="joint_a3" value="1.1248573201676095" />
		<joint name="joint_a4" value="-0.014224119244488426" />
		<joint name="joint_a5" value="1.309681509488257" />
		<joint name="joint_a6" value="0.0" />
	</group_state>
	<group_state name="armPlaceDetectPosePos" group="leapting_arm">
		<joint name="joint_a1" value="1.523314" />
		<joint name="joint_a2" value="-1.579361844258" />
		<joint name="joint_a3" value="1.12114781482" />
		<joint name="joint_a4" value="0.006897768096684" />
		<joint name="joint_a5" value="1.62371093980" />
		<joint name="joint_a6" value="-1.66032628822" />
	</group_state>
	<group_state name="armPickupMidPosePos" group="leapting_arm">
		<joint name="joint_a1" value="-0.30181207999999" />
		<joint name="joint_a2" value="-1.44733110" />
		<joint name="joint_a3" value="1.639219586" />
		<joint name="joint_a4" value="-0.01607554706171" />
		<joint name="joint_a5" value="1.499758" />
		<joint name="joint_a6" value="0.154819815" />
	</group_state>
	<group_state name="armPlaceDetect" group="leapting_arm">
		<joint name="joint_a1" value="-1.571915022" />
		<joint name="joint_a2" value="-0.8628964" />
		<joint name="joint_a3" value="0.9292418" />
		<joint name="joint_a4" value="-0.0322861" />
		<joint name="joint_a5" value="1.50485485" />
		<joint name="joint_a6" value="-1.5414852" />
	</group_state>
	<group_state name="armPickupMidPose" group="leapting_arm">
		<joint name="joint_a1" value="0.30181207999999" />
		<joint name="joint_a2" value="-1.44733110" />
		<joint name="joint_a3" value="1.639219586" />
		<joint name="joint_a4" value="-0.01607554706171" />
		<joint name="joint_a5" value="1.499758" />
		<joint name="joint_a6" value="0.154819815" />
	</group_state>
	<group_state name="armPlaceMidPose" group="leapting_arm">
		<joint name="joint_a1" value="-1.8506205" />
		<joint name="joint_a2" value="-1.352495" />
		<joint name="joint_a3" value="1.191933351" />
		<joint name="joint_a4" value="-0.014217813" />
		<joint name="joint_a5" value="1.30573669" />
		<joint name="joint_a6" value="-1.8975896" />
	</group_state>
	<group_state name="calibration_1" group="leapting_arm">
		<joint name="joint_a1" value="-0.3180175949231931" />
		<joint name="joint_a2" value="-1.0050600845189912" />
		<joint name="joint_a3" value="1.5577383143835428" />
		<joint name="joint_a4" value="0.10885477173019223" />
		<joint name="joint_a5" value="0.9412201118179508" />
		<joint name="joint_a6" value="-1.9134295742493195" />
	</group_state>
	<group_state name="calibration_2" group="leapting_arm">
		<joint name="joint_a1" value="6.874851923605663e-05" />
		<joint name="joint_a2" value="-0.7221913612424464" />
		<joint name="joint_a3" value="1.029950382061315" />
		<joint name="joint_a4" value="-0.016110471100043898" />
		<joint name="joint_a5" value="1.0808462599912794" />
		<joint name="joint_a6" value="-3.12413936106985" />
	</group_state>
	<group_state name="calibration_3" group="leapting_arm">
		<joint name="joint_a1" value="6.89754120388159e-05" />
		<joint name="joint_a2" value="-1.2879293313943114" />
		<joint name="joint_a3" value="2.047826401824408" />
		<joint name="joint_a4" value="-0.01611057581979902" />
		<joint name="joint_a5" value="0.9063133347918465" />
		<joint name="joint_a6" value="5.235987755982989e-08" />
	</group_state>
	<group_state name="calibration_4" group="leapting_arm">
		<joint name="joint_a1" value="0.25453764874025414" />
		<joint name="joint_a2" value="-0.910769998130229" />
		<joint name="joint_a3" value="1.4446406472417526" />
		<joint name="joint_a4" value="0.04637224630818302" />
		<joint name="joint_a5" value="0.9412200594580733" />
		<joint name="joint_a6" value="1.8744841751472705" />
	</group_state>
	<group_state name="calibration_5" group="leapting_arm">
		<joint name="joint_a1" value="6.867870606597687e-05" />
		<joint name="joint_a2" value="-0.887197965225229" />
		<joint name="joint_a3" value="1.25614491349344" />
		<joint name="joint_a4" value="0.04637215904172042" />
		<joint name="joint_a5" value="1.883697785721841" />
		<joint name="joint_a6" value="0.06248285703456708" />
	</group_state>
	<group_state name="calibration_6" group="leapting_arm">
		<joint name="joint_a1" value="0.12730319862974512" />
		<joint name="joint_a2" value="-0.8871977034258411" />
		<joint name="joint_a3" value="1.3315433989789827" />
		<joint name="joint_a4" value="-0.26604210867782246" />
		<joint name="joint_a5" value="1.883697785721841" />
		<joint name="joint_a6" value="-0.12496538245657628" />
	</group_state>
	<group_state name="calibration_7" group="leapting_arm">
		<joint name="joint_a1" value="0.2545377360067168" />
		<joint name="joint_a2" value="-0.9107703297427869" />
		<joint name="joint_a3" value="1.3692423013825499" />
		<joint name="joint_a4" value="-0.640938378220599" />
		<joint name="joint_a5" value="1.7440717073616825" />
		<joint name="joint_a6" value="-0.3748965138888717" />
	</group_state>
	<group_state name="calibration_8" group="leapting_arm">
		<joint name="joint_a1" value="0.2545380327126896" />
		<joint name="joint_a2" value="-1.0522051626199573" />
		<joint name="joint_a3" value="1.52003874875486" />
		<joint name="joint_a4" value="-0.765904092289733" />
		<joint name="joint_a5" value="1.5346319353229751" />
		<joint name="joint_a6" value="-0.8747591432726055" />
	</group_state>
	<group_state name="calibration_9" group="leapting_arm">
		<joint name="joint_a1" value="0.2545377883665943" />
		<joint name="joint_a2" value="-1.1464943065309232" />
		<joint name="joint_a3" value="1.7085344825031727" />
		<joint name="joint_a4" value="-1.0783177840506226" />
		<joint name="joint_a5" value="0.9412200594580733" />
		<joint name="joint_a6" value="-1.1246899430923432" />
	</group_state>
	<group_state name="calibration_10" group="leapting_arm">
		<joint name="joint_a1" value="0.19092041495835338" />
		<joint name="joint_a2" value="-1.1936395940713997" />
		<joint name="joint_a3" value="1.8216317482192352" />
		<joint name="joint_a4" value="-1.2032838297323147" />
		<joint name="joint_a5" value="0.8365001647120733" />
		<joint name="joint_a6" value="-1.3121381302236088" />
	</group_state>
	<group_state name="calibration_11" group="leapting_arm">
		<joint name="joint_a1" value="0.19092050222481596" />
		<joint name="joint_a2" value="-1.2407842009334675" />
		<joint name="joint_a3" value="1.859330929875483" />
		<joint name="joint_a4" value="-1.2657669136596845" />
		<joint name="joint_a5" value="0.6619670300731302" />
		<joint name="joint_a6" value="-1.437103774479573" />
	</group_state>
	<group_state name="calibration_12" group="leapting_arm">
		<joint name="joint_a1" value="0.12730319862974512" />
		<joint name="joint_a2" value="-1.2172116269763995" />
		<joint name="joint_a3" value="1.8970302337047782" />
		<joint name="joint_a4" value="-1.3907317550641927" />
		<joint name="joint_a5" value="0.3827147860863504" />
		<joint name="joint_a6" value="-1.562069942334313" />
	</group_state>
	<group_state name="calibration_13" group="leapting_arm">
		<joint name="joint_a1" value="-0.06354832818303205" />
		<joint name="joint_a2" value="-1.2172114873500592" />
		<joint name="joint_a3" value="1.8593310520485304" />
		<joint name="joint_a4" value="-1.5781799421954585" />
		<joint name="joint_a5" value="-0.42013675709750375" />
		<joint name="joint_a6" value="-1.8744829708700868" />
	</group_state>
	<group_state name="calibration_14" group="leapting_arm">
		<joint name="joint_a1" value="-0.0635485376225423" />
		<joint name="joint_a2" value="-1.2407842009334675" />
		<joint name="joint_a3" value="1.89702983227905" />
		<joint name="joint_a4" value="-1.7656285307524522" />
		<joint name="joint_a5" value="-0.5248567216566737" />
		<joint name="joint_a6" value="-1.9369664562231843" />
	</group_state>
	<group_state name="calibration_15" group="leapting_arm">
		<joint name="joint_a1" value="-0.12716577140444307" />
		<joint name="joint_a2" value="-1.1936390530193315" />
		<joint name="joint_a3" value="1.8216320100186232" />
		<joint name="joint_a4" value="-1.8281112132540942" />
		<joint name="joint_a5" value="-0.7342960224564831" />
		<joint name="joint_a6" value="-1.9994486151260507" />
	</group_state>
	<group_state name="calibration_16" group="leapting_arm">
		<joint name="joint_a1" value="-0.0635485376225423" />
		<joint name="joint_a2" value="-1.075777195524957" />
		<joint name="joint_a3" value="1.6331361366439703" />
		<joint name="joint_a4" value="-2.3904565949526404" />
		<joint name="joint_a5" value="-1.0833622219211994" />
		<joint name="joint_a6" value="-2.1244143815550625" />
	</group_state>
	<group_state name="calibration_17" group="leapting_arm">
		<joint name="joint_a1" value="-0.0635486248890049" />
		<joint name="joint_a2" value="-1.0522052324331272" />
		<joint name="joint_a3" value="1.5577381747572028" />
		<joint name="joint_a4" value="-2.3279740346240456" />
		<joint name="joint_a5" value="-1.2229881606550177" />
		<joint name="joint_a6" value="-2.0619315768803728" />
	</group_state>
	<group_state name="calibration_18" group="leapting_arm">
		<joint name="joint_a1" value="-0.1271656841379805" />
		<joint name="joint_a2" value="-0.9579152158575353" />
		<joint name="joint_a3" value="1.4069417448381853" />
		<joint name="joint_a4" value="-2.5154222217553115" />
		<joint name="joint_a5" value="-1.3975210858544507" />
		<joint name="joint_a6" value="-2.124413857956287" />
	</group_state>
	<group_state name="calibration_19" group="leapting_arm">
		<joint name="joint_a1" value="-0.0635485376225423" />
		<joint name="joint_a2" value="-0.8400530791104805" />
		<joint name="joint_a3" value="1.3315438004047104" />
		<joint name="joint_a4" value="-2.6403876042118877" />
		<joint name="joint_a5" value="-1.537147146761317" />
		<joint name="joint_a6" value="-2.37434553044065" />
	</group_state>
	<group_state name="calibration_20" group="leapting_arm">
		<joint name="joint_a1" value="6.831218692305805e-05" />
		<joint name="joint_a2" value="-0.8400529569374331" />
		<joint name="joint_a3" value="1.2561450356664874" />
		<joint name="joint_a4" value="-2.8278357913431535" />
		<joint name="joint_a5" value="-1.5707960649955086" />
		<joint name="joint_a6" value="-2.561793996824597" />
	</group_state>
	<group_state name="calibration_21" group="leapting_arm">
		<joint name="joint_a1" value="6.890559886873613e-05" />
		<joint name="joint_a2" value="-0.9107702075697393" />
		<joint name="joint_a3" value="1.4446406472417526" />
		<joint name="joint_a4" value="-3.06165627714248" />
		<joint name="joint_a5" value="-1.5707964664212366" />
		<joint name="joint_a6" value="-3.06165627714248" />
	</group_state>
	<group_state name="armPickDetect" group="leapting_arm">
		<joint name="joint_a1" value="0.0230037" />
		<joint name="joint_a2" value="-0.7849" />
		<joint name="joint_a3" value="1.5625" />
		<joint name="joint_a4" value="-0.03018" />
		<joint name="joint_a5" value="0.8006" />
		<joint name="joint_a6" value="0.0323" />
	</group_state>
	<virtual_joint name="virtual_joint" type="floating" parent_frame="base_arm"
		child_link="base_link" />
	<disable_collisions link1="base_arm" link2="link_1" reason="Adjacent" />
	<disable_collisions link1="base_arm" link2="link_2" reason="Never" />
	<disable_collisions link1="base_arm" link2="link_3" reason="Never" />
	<disable_collisions link1="base_arm" link2="base_car" reason="Never" />
	<disable_collisions link1="base_car" link2="base_yuntai" reason="Never" />
	<disable_collisions link1="link_1" link2="link_2" reason="Adjacent" />
	<disable_collisions link1="link_1" link2="link_3" reason="Never" />
	<disable_collisions link1="link_2" link2="link_3" reason="Adjacent" />
	<disable_collisions link1="link_2" link2="link_4" reason="Never" />
	<disable_collisions link1="link_3" link2="link_4" reason="Adjacent" />
	<disable_collisions link1="link_3" link2="link_5" reason="Never" />
	<disable_collisions link1="link_3" link2="link_6" reason="Never" />
	<disable_collisions link1="link_4" link2="link_5" reason="Adjacent" />
	<disable_collisions link1="link_4" link2="link_6" reason="Never" />
	<disable_collisions link1="link_5" link2="link_6" reason="Adjacent" />
	<disable_collisions link1="link_5" link2="tool0" reason="Never" />
	<disable_collisions link1="link_6" link2="tool0" reason="Adjacent" />
	<group_state name="test12043" group="leapting_arm">
		<joint name="joint_a1" value="-1.107297" />
		<joint name="joint_a2" value="-1.323634" />
		<joint name="joint_a3" value="1.124829" />
		<joint name="joint_a4" value="-0.014191" />
		<joint name="joint_a5" value="1.309625" />
		<joint name="joint_a6" value="7.3e-05" />
	</group_state>
	<group_state name="test12046" group="leapting_arm">
		<joint name="joint_a1" value="-1.107297" />
		<joint name="joint_a2" value="-1.323634" />
		<joint name="joint_a3" value="1.124829" />
		<joint name="joint_a4" value="-0.014191" />
		<joint name="joint_a5" value="1.309625" />
		<joint name="joint_a6" value="7.3e-05" />
	</group_state>
	<group_state name="test12048" group="leapting_arm">
		<joint name="joint_a1" value="-1.107297" />
		<joint name="joint_a2" value="-1.323634" />
		<joint name="joint_a3" value="1.124829" />
		<joint name="joint_a4" value="-0.014191" />
		<joint name="joint_a5" value="1.309625" />
		<joint name="joint_a6" value="7.3e-05" />
	</group_state>
</robot>
